changelog/@unreleased/pr-1824.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Add Gradle 7 support
```

Comment:
Do we require a new minimum gradle version now? If so, should probably make it a break.

Reply:
If we decide with [this one](https://github.com/palantir/gradle-baseline/pull/1824/files#r670421739) then we would require Gradle 6.7.

If we avoid this compile dependency e.g. through reflection, I don't think we introduce a new min version.